### PR TITLE
Remote control updates

### PR DESCRIPTION
**Pull request by amelchio**
**Description:**

* Remove obsolete mentions of `sync` (moved to Harmony) and `device` (is now optional) from remote, remote.apple_tv and remote.itach.
* Document all service data attributes for Harmony remote.
* Cut down on complex Harmony examples that confuse rather than help.

## Checklist:

  - [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by amelchio** [comment]
I know it feels bad to remove a big chunk of work but in this context I find it confusing. I believe the platform documentation should explain the platform and not teach how to combine things, we have other resources for that.

I considered moving it to the Cookbook but there are already similar examples, [for example here](https://home-assistant.io/cookbook/perform_actions_based_on_input_select/).

File: source/_components/remote.harmony.markdown [hunk 1]
```diff
@@ -64,32 +64,58 @@ Upon startup one file will be written to your Home Assistant configuration direc
...
 
-- **Turn Off**: Turn off all devices that were switched on from the start of the current activity.s
-- **Turn On**: Start an activity, will start the default activity from configuration.yaml if no activity is specified.  The specified activity can either be the activity name or the activity ID from the configuration file written to your [Home Assistant configuration directory](/docs/configuration/).
-- **Send Command**: Send a single command or a set of commands to one device, device ID and available commands are written to the configuration file at startup. You can optionally specify the number of times you wish to repeat the command(s) and delay you want between repeated command(s).
-- **Sync**: Synchronizes the Harmony device with the Harmony web service if any changes are made from the web portal or app.
+Turn off all devices that were switched on from the start of the current activity.
 
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      yes | Only act on specific remote. Else targets all.
```
Comment (frenck) [inline source/_components/remote.harmony.markdown]: Missing determiner: `on a specific remote`
Use comma: `remote, else target all`

File: source/_components/remote.harmony.markdown [hunk 2]
```diff
@@ -64,32 +64,58 @@ Upon startup one file will be written to your Home Assistant configuration direc
...
 
-### {% linkable_title Examples %}
+### {% linkable_title Service `remote.turn_on` %}
+
+Start an activity. Will start the default `activity` from configuration.yaml if no activity is specified.  The specified activity can either be the activity name or the activity ID from the configuration file written to your [Home Assistant configuration directory](/docs/configuration/).
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      yes | Only act on specific remote. Else targets all.
```
Comment (frenck) [inline source/_components/remote.harmony.markdown]: Missing determiner: `on a specific remote`
Use comma: `remote, else target all`

File: source/_components/remote.harmony.markdown [hunk 3]
```diff
@@ -64,32 +64,58 @@ Upon startup one file will be written to your Home Assistant configuration direc
...
+| `activity`             |      yes | Activity ID or Activity Name to start.
+
+### {% linkable_title Service `remote.send_command` %}
+
+Send a single command or a set of commands to one device, device ID and available commands are written to the configuration file at startup. You can optionally specify the number of times you wish to repeat the command(s) and delay you want between repeated command(s).
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      yes | Only act on specific remote. Else targets all.
```
Comment (frenck) [inline source/_components/remote.harmony.markdown]: Missing determiner: `on a specific remote`
Use comma: `remote, else target all`

File: source/_components/remote.harmony.markdown [hunk 4]
```diff
@@ -64,32 +64,58 @@ Upon startup one file will be written to your Home Assistant configuration direc
...
+| `device`               |       no | Device ID to send command to.
```
Comment (frenck) [inline source/_components/remote.harmony.markdown]: Missing determiner: `to send the command to`

File: source/_components/remote.harmony.markdown [hunk 5]
```diff
@@ -64,32 +64,58 @@ Upon startup one file will be written to your Home Assistant configuration direc
...
+  delay_secs: 0.6
 ```
 
+### {% linkable_title Service `remote.harmony_sync` %}
+
+Synchronize the Harmony device with the Harmony web service if any changes are made from the web portal or app.
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            |      yes | Only act on specific remote. Else targets all.
```
Comment (frenck) [inline source/_components/remote.harmony.markdown]: Missing determiner: `on a specific remote`
Use comma: `remote, else target all`

**Comment by frenck** [comment]
@amelchio Reviewed your PR and found a couple of tiny spelling/grammar things. Could you please take a look?

**Comment by amelchio** [comment]
@frenck Thanks. I fixed those now, also in the component doc where I had copied it from :)

**Comment by frenck** [comment]
@amelchio Even better! Thank you for this 🏅

**Pull request merged**